models/Participant.model.js

```diff
@@ -1,13 +1,17 @@
 const mongoose = require("mongoose");
 const Schema = mongoose.Schema;
 
-const participantSchema = Schema(
-  {
+const participantSchema = Schema({
+  userId: Number,
+  actionType: ["receiver", "provider", "driver", "loaner", "borrower"],
+  endCoords: {
```

Comment:
Can we change this to be `endLoc` because it includes information other than coords?